### PR TITLE
Fix all hardcoded strings

### LINKED ISSUE #1030: Fix all hardcoded strings
## Description
Fix all hardcoded strings in strings.xml

### Enhancement to Update [Optional]
- [x] Some strings are still in hardcoded, so fix them all.

## Definition of Done
- [x] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hours 

@MeepyMay @Rimjhim28 @prachi1210  I want work on this enhancement!

### PR DESCRIPTION
**Pull request by pprathameshmore**
### Description
Fix all hardcoded strings in XML files.

Fixes #1030 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml [hunk 1]
```diff
@@ -4,80 +4,85 @@
...
         android:gravity="center"
+        android:text="@string/zero_text"
         android:textSize="25sp"
-        android:textStyle="bold"/>
+        android:textStyle="bold" />
+
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:baselineAligned="false"
```
Comment (Rimjhim28) [inline PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml]: This does not concerns with removing hard coded text. Revert this.

Comment (pprathameshmore) [reply PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml]: What name should I give for this commit? @Rimjhim28

Comment (Rimjhim28) [reply PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml]: @pprathameshmore Depends on your perception. Maintain 1 commit per PR.

Comment (pprathameshmore) [reply PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml]: @Rimjhim28 I made changes as per request! Please review! 👍

**Comment by pprathameshmore** [comment]
@Rimjhim28 I am not getting!

**Comment by Rimjhim28** [comment]
Agreed @geekanamika! I am closing the PR. I will create a new issue and make it available.

**Pull request closed without merging**